src/features/people/ActorTile/styled.js

```diff
@@ -6,28 +6,33 @@ const bpMobile = ({ theme }) => theme.breakpoint.mobile;
 
 export const StyledActorTile = styled(Link)`
   display: block;
-  height: 339px;
-  width: 208px;
   padding: 16px;
   text-align: center;
   text-decoration: none;
   background-color: ${({ theme }) => theme.color.white};
+  transition: all 170ms cubic-bezier(0.45, 0.05, 0.55, 0.95);
+
+  &:hover {
+    transform: translateY(-4px);
+    box-shadow: 0px 6px 18px 2px #ced7d9;
```

Comment:
Can we make that shadow more closer to  #6195ED ( cornflower blue ) ???

But overal its, awesome effect :)